jetty-runner/src/main/java/org/eclipse/jetty/runner/Runner.java

```diff
@@ -63,7 +63,9 @@
  * Runner
  * <p>
  * Combine jetty classes into a single executable jar and run webapps based on the args to it.
+ * @deprecated No replacement provided or available.  Migrate to jetty-home (and use {@code ${jetty.base}} directory).
```

Comment:
I don't think anyone using the jetty-runner is going to want to use a full distro installation. Maybe direct them to the maven jetty plugin for development and then tell them to build their own executable jar.  We should probably have some instructions on how to do the latter, enough people seem to use jetty that way :)

Reply:
that's the embedded-jetty way, yeah, and its an easy enough way to convince people.
and the uber-jar way (a custom jetty)
and the live-war way (another custom jetty)
there's also the spark way. (a subset of jetty)
and the dropwizard way. (a subset of jetty)